makefile

```diff
@@ -232,6 +232,8 @@ $(buildDir)/dist.tar.gz:$(buildDir)/make-tarball $(clientBinaries) $(uiFiles)
 	./$< --name $@ --prefix $(name) $(foreach item,$(distContents),--item $(item)) --exclude "public/node_modules" --exclude "clients/.cache"
 # end main build
 
+update-lobster: 
+	EVGHOME=$(shell pwd) ./buildscripts/update-lobster.sh
```

Comment:
the EVGHOME part isn't needed since the script doesn't use that variable

Reply:
I suggested using it as a guard against invoking update-lobster.sh from any random place in the filesystem.